framework/src/transfers/MultiAppFieldTransfer.C

```diff
@@ -23,8 +23,7 @@ validParams<MultiAppFieldTransfer>()
   InputParameters params = validParams<MultiAppTransfer>();
   params.addRequiredParam<std::vector<AuxVariableName>>(
       "variable", "The auxiliary variable to store the transferred values in.");
-  params.addRequiredParam<std::vector<VariableName>>("source_variable",
-                                                     "The variable to transfer from.");
+  params.addParam<std::vector<VariableName>>("source_variable", "The variable to transfer from.");
```

Comment:
This makes the line in the ctor that retrieves these vectors unsafe since they can be omitted completely now. Did you mean to make the source_variable(s) optional?